### ISSUE TITLE
Object is not iterable error when result set is empty

### ISSUE THREAD

**Issue by albertocalderari** [doitintl/bigquery-grafana#316]
## Bug Report
A select query returning no results triggers an exception rather than just an empty result.
This behaviour does not appear when using a different data source which leads to an empty `series` object (i.e. prometheus)
<img width="929" alt="Screenshot 2021-03-16 at 13 17 52" src="https://user-images.githubusercontent.com/7837183/111315187-0a204e00-865a-11eb-9205-b421cab78af8.png">

## Expected Behaviour
The chart should display "No data" or no display anything

## Actual Behaviour
An Object is not iterable exception is thrown

## Steps to Reproduce the Problem

1. create a new panel
2. using the BQ source create a query that returns no results:
i.e. ```SELECT
 $__timeGroupAlias(TimeStamp,1h),
  count(*) AS errors
from table
where 1=2
```

## Specifications

- Version: 2.0.1
- Platform: MacOs 10.15.7
- Grafana Version: Grafana v7.2.1 (72a6c64532)

**Comment by ofir5300** [comment]
Hi @albertocalderari ,
Thanks for sharing this great input, I will probably fix it in the coming release (2.0.2)

**Comment by albertocalderari** [comment]
NP, looking forward to the new release!
